### PR TITLE
Update link-type.md

### PR DESCRIPTION
**Pull request by cmanhard**
Improve description of links

### CODE REVIEW

File: documentation/data/link-type.md [hunk 1]
```diff
@@ -5,7 +5,16 @@ uid: linkType
 # Link Data Messages
 
 
-A Link is used to create relationships between Types and instances of Types, and is defined using the pre-defined typeid `__Link` in the Data Message.
+A Link is used to create relationships between omf created resources.  Links are defined using the pre-defined typeid `__Link` in the Data Message.
```
Comment (zryska) [inline documentation/data/link-type.md:8]: Extremely minor - we might capitalizing OMF.

File: documentation/data/link-type.md [hunk 2]
```diff
@@ -5,7 +5,16 @@ uid: linkType
 # Link Data Messages
 
 
-A Link is used to create relationships between Types and instances of Types, and is defined using the pre-defined typeid `__Link` in the Data Message.
+A Link is used to create relationships between omf created resources.  Links are defined using the pre-defined typeid `__Link` in the Data Message. 
+For example:
+ - links between static instances, such as a plant to a tank, or a tank to a pump
+ - links from static instances to containers, such as a tank to a measurement
+ - links from a specific property name in a static instances to a measurement, such as a tank inflow property to a measurement
```
Comment (zryska) [inline documentation/data/link-type.md:12]: use singular for the instance? => a specific property name in a static instance to a measurement

**Review by zryska — APPROVED** [review]

File: documentation/data/link-type.md (hunk 1 above)
Comment (cmanhard) [reply documentation/data/link-type.md]: Done

File: documentation/data/link-type.md (hunk 2 above)
Comment (cmanhard) [reply documentation/data/link-type.md]: Thanks.  Done.

**Pull request merged**